pipeline/cip_crosswalk.py

```diff
@@ -113,12 +169,51 @@ def _load_1990_to_2000_crosswalk(self):
         df = df[df[origin_col] != df[destination_col]]
         cip_map = dict(zip(df[origin_col], df[destination_col]))
         cip_code_to_title_map = dict(zip(df[destination_col], df[destination_title_col]))
-
+        
         cip_map.update(self.CORRECTIONS_1990)
         mappers = {"cip_map": cip_map, "title_map": cip_code_to_title_map}
 
         self.crosswalk[year_range] = mappers
 
+    def _load_1985_to_1990_crosswalk(self):
```

Comment:
Something not quite right is happening with the mapping to cip2020 - I end up with string title values in the cip2020 column when I expect 7-digit CIPs.

One way to try to figure out what's going wrong is to see where any values in the mapping have more than 7 characters.

Reply:
hm, i'm not seeing this issue. it was a problem at first but i fixed it, and now if i run the debugger i'm getting that everything in the cip mapping has 7 characters.

Reply:
I added a test for this